pkg/cmd/init.go

```diff
@@ -172,6 +172,17 @@ func initCerts(cfg *config.MicroshiftConfig) ([]byte, *cryptomaterial.Certificat
 			"service-ca",
 			cryptomaterial.ServiceCADir(certsDir),
 			cryptomaterial.ServiceCAValidityDays,
+		).WithServingCertificates(
+			&cryptomaterial.ServingCertificateSigningRequestInfo{
+				CertificateSigningRequestInfo: cryptomaterial.CertificateSigningRequestInfo{
+					Name:         "openshift-controller-manager-serving",
+					ValidityDays: 730,
```

Comment:
Why not adding a constant as the rest of validity days ones? or just using the `ServiceCAServingCertValidityDays`?

Reply:
Turns out such a constant already exists, only I forgot to use it :+1: